### PR TITLE
Add AdvancedNetworkStatsDTO to TimedMemberState

### PR DESCRIPTION
**Pull request by puzpuzpuz**
* Partial forward-port of #15541
* Adds `AdvancedNetworkStatsDTO` to `TimedMemberState`, as MC 4.0 will be consuming `TimedMemberState`

MC counterpart PR: https://github.com/hazelcast/management-center/pull/2232

### CODE REVIEW

**Comment by puzpuzpuz** [comment]
run-lab-run

**Comment by puzpuzpuz** [comment]
run-lab-run

**Comment by puzpuzpuz** [comment]
run-lab-run

File: hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java [hunk 1]
```diff
@@ -223,7 +232,7 @@ private void createClusterHotRestartStatus(MemberStateImpl memberState) {
         memberState.setClusterHotRestartStatus(state);
     }
 
-    protected void createNodeState(MemberStateImpl memberState) {
+    private void createNodeState(MemberStateImpl memberState) {
```
[the remark this answers is not in the document]
Comment (mmedenjak) [reply hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java]: Created a [PR](https://github.com/hazelcast/hazelcast/pull/15893), please take a look.

[the remark this answers is not in the document]
Comment (emre-aydin) [reply hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java]: @mmedenjak, Andrey is on holiday. We can change it back in another PR, I guess Andrey just had OS project open in his IDE and applied an autofix from it. We should prioritize having "hazel the octopus" live.

[the remark this answers is not in the document]
Comment (puzpuzpuz) [reply hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java]: @mmedenjak thanks for noticing this and for creating the fix PR. I missed the fact that it was made `protected` on purpose.

**Pull request merged**